src/aff_expr.jl

```diff
@@ -261,8 +261,15 @@ Replaces `getvalue` for most use cases.
 """
 value(a::GenericAffExpr) = value(a, value)
 
+function verify_ownership(model::AbstractModel, a::GenericAffExpr)
+    for variable in keys(a.terms)
+        verify_ownership(model, variable)
+    end
+end
+
 # Note: No validation is performed that the variables in the AffExpr belong to
-# the same model.
+# the same model. The verification is done in `verify_ownership` which should be
+# called before calling `moi_function`.
```

Comment:
The comment is confusing because this function is `MOI.ScalarAffineFunction`, not `moi_function`.